README.md

```diff
@@ -196,7 +197,19 @@ The output will be:
 
     Result: Hello, my-name!
 
-## Usage
+## Next Steps
+Temporal can be implemented in your code in many different ways, to suit your applicaiton's needs. The links below will give you much more information about how Temporal works with Python:
+
+* [Code Samples](https://github.com/temporalio/samples-python) - if you want to start with some code, we have provided somepre-built samples
+* [Application Development Guide](https://docs.temporal.io/application-development?lang=python) - Our Python specific Developer's Guide will give you much more information on how to build with Temporal in your Python applications than our SDK README ever could (or should).
```

Comment:
You are inconsistent with whether you have trailing punctuation at the end of these phrases after `-`

Reply:
ty!